package.json

```diff
@@ -67,5 +67,8 @@
       "@typescript-eslint/explicit-function-return-type": 0,
       "@typescript-eslint/no-explicit-any": 0
     }
+  },
+  "dependencies": {
+    "web-streams-polyfill": "^2.0.4"
```

Comment:
Can we use this package? https://www.npmjs.com/package/node-web-streams
Looks like it can convert streams.

Reply:
I have decided to implement the functions based on this package solution because I wanted to reduce the dependencies as much as possible. It has only 2 for now, but you never know what will happen in the future. Also, I needed some extra utilities.